bin/create-release-tag

```diff
@@ -91,10 +91,10 @@ bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 tmp=$(. "$bindir"/_release.sh; extract_release_notes)
 
 # Create a signed tag with the commit message.
```

Comment:
should "signed" be removed from this comment?

Reply:
Right, thanks!